### PR TITLE
Account purge

### PR DESCRIPTION
**Pull request by audip**
- Add purge-accounts job to orbit
- Add feed-reader job to orbit
- Rebased to remove zone commits

### CODE REVIEW

**Comment by corystone** [comment]
Still has all the zone stuff. I think you squashed the zone stuff in your first commit.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
-####
+
+#Lunr 
 
 Lunr plans to be an OpenStack compatible volume manager.
 
-==============
-Geting started
-==============
+##Geting started
```
Comment (buhman) [inline README.md]: why was this converted back to markdown?

File: lunr/common/purge.py [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+            except BadRequest:
+                if attempts > 0:
+                    raise
+                self.clean_up_volume(volume)
+                attempts += 1
+                continue
+
+    def _delete_volume(self, volume):
+        # Skip volumes in strange status
+        if volume['status'] in ('NEW', 'DELETED', 'ERROR', 'DELETING'):
```
Comment (buhman) [inline lunr/common/purge.py]: I think we should only be passing things to `_delete_volume` that we actually think should be deleted.

File: lunr/db/migrations/versions/008_add_terminator_tables.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    Column('event_id', String(45), unique=True, nullable=False),
+    Column('tenant_id', String(20), index=True, nullable=False),
+    Column('timestamp', DateTime, nullable=False),
+    Column('processed', String(5), nullable=False),
+    Column('last_purged', DateTime, nullable=True),
+    Column('created_at', DateTime),
+    Column('last_modified', DateTime),
+    **table_kwargs
+)
+
```
Comment (buhman) [inline lunr/db/migrations/versions/008_add_terminator_tables.py]: Some light high-level documentation on what these tables do would be useful for reviewing this.

Comment (audip) [inline lunr/db/migrations/versions/008_add_terminator_tables.py]: Added light doc for tables

**Comment by audip** [comment]
@corystone I am not sure I can see zone stuff in codebase now, can you check and comment?

File: lunr/orbit/jobs/purge_accounts.py [hunk 4]
```diff
@@ -108,8 +108,8 @@ def collect_totals(self, purger):
         self.total['backups'] += purger.total['backups']
         self.total['backup-size'] += purger.total['backup-size']
         print(repr(purger.total['vtypes']))
-        for key in purger.total['vtypes'].keys():
-            self.total['vtypes'][key] += purger.total['vtypes'][key]
+        # for key in purger.total['vtypes'].keys():
```
Comment (audip) [inline lunr/orbit/jobs/purge_accounts.py]: @thrawn01 it's unable to find keys in vtypes, need help in fixing this.

**Pull request closed without merging**